daml-lf/archive/BUILD.bazel

```diff
@@ -128,6 +128,7 @@ da_scala_test_suite(
         ":DarReaderTest.dalf",
         ":DarReaderTest.dar",
         ":daml_lf_1.11_archive_proto_srcs",
+        ":daml_lf_1.13_archive_proto_srcs",
```

Comment:
bazel is probably going to yell at you if you don’t sort ths and even if not, sorting is probably a good idea.

Reply:
fixed.